src/main/java/com/arpnetworking/tsdcore/statistics/StatisticFactory.java

```diff
@@ -15,14 +15,14 @@
  */
 package com.arpnetworking.tsdcore.statistics;
 
+import com.arpnetworking.steno.Logger;
+import com.arpnetworking.steno.LoggerFactory;
```

Comment:
You need to change the LOGGER calls in this class for them to actually log anything.

Reply:
The LogbackSteno logger is backwards compatible. This should work as-is. Although it would be better if there were enhance log statements.